lib/raop.h

```diff
@@ -28,19 +28,22 @@ extern "C" {
 
 
 typedef struct raop_s raop_t;
+typedef struct raop_conn_s raop_conn_t;
```

Comment:
For implementing background drawing according to connections, I added two callbacks(conn_init and conn_destroy) to listen connections activity. 

These two callbacks maybe is not only for background drawing, it can be do more things, and it should have a connection  related parameter. So I exposed the `raop_conn_t`.

But in terms of drawing the background,  it's not necessary.

What do you thing?

Reply:
It is used in new added callbacks `conn_init` and `conn_destory` for generic usage. In terms of drawing background, it is not necessary. 

What do you think?

Reply:
Oh, I see, this is just the typedef. At first glance, I thought it was a global variable. Still, just for the background rendering, I'd prefer not to expose library internals. If we need it at a later point, we can still add it back in pretty easily. So, I'd suggest to remove the raop_conn_t argument from the two new callback functions.

Reply:
OK

Reply:
Removed  the `conn` parameter from `conn_init` and `conn_destroy` and the conn typedef in the latest commit